app/templates/categories.hbs

```diff
@@ -60,3 +60,11 @@
 </div>
 
 <Pagination @pages={{this.pages}} @prevPage={{this.prevPage}} @nextPage={{this.nextPage}} />
+
+<div class='categories-footer'>
+  Got an idea for another category?
+  <a href='https://github.com/rust-lang/crates.io/blob/master/src/boot/categories.toml#L1-L3'>Suggest it!</a>
+  &bull;
```

Comment:
What do you think about removing this suggestion and link?  We briefly discussed the change in our team meeting yesterday, and we're not sure that we want to include this part yet.  We're generally cautious of adding new categories since we can't later remove them, and such PRs tend to take a long time to review which it isn't a great developer experience for a first-time contributor.

Reply:
No problem, I removed the link.

I was just going off the suggestion in the original issue, but that was written a while ago. I do think it'd be nice to link to the config as a sort of "this is where these are defined" messaging, but since categories aren't removable, it makes sense to be a bit cautious about pointing people to that file. This'll work fine, I think.